src/current/v23.2/known-limitations.md

```diff
@@ -10,100 +10,61 @@ This page describes newly identified limitations in the CockroachDB {{page.relea
+  - The `INTO` statement in PL/pgSQL does not support the `STRICT` option. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/107854)
 
-### Limitations for composite types
+#### Type Handling and Variable Declarations
 
-- {% include {{page.version.version}}/cdc/types-udt-composite-general.md %} The following limitations apply:
-    - {% include {{page.version.version}}/cdc/avro-udt-composite.md %}
-    - {% include {{page.version.version}}/cdc/csv-udt-composite.md %}
+- There are several known limitations regarding type handling and variable declarations in the current implementation of PL/pgSQL:
+  - The language does not support `RECORD` input arguments in UDFs. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/105713)
+  - Type coercion does not behave consistently with PostgreSQL, particularly when values are too wide for the target type. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/115385)
+  - PL/pgSQL lacks support for `RECORD`-type variables, which are dynamically typed in PostgreSQL. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114874)
+  - Syntax for accessing members of composite types without parentheses is not supported. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/114687)
+  - There is no support for adding context to errors with `RAISE` and for declaring variable collation. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/106237)
+  - PL/pgSQL does not yet support `NOT NULL` variable declarations or set-returning functions. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/105243)
 
-### Common table expressions are not supported in user-defined functions
+#### Limitations in User-Defined Functions (UDFs)
 
-[Common table expressions]({% link {{ page.version.version }}/common-table-expressions.md %}) (CTE), recursive or non-recursive, are not supported in [user-defined functions]({% link {{ page.version.version }}/user-defined-functions.md %}) (UDF). That is, you cannot use a `WITH` clause in the body of a UDF.
+  - The `PERFORM` keyword is not supported in UDFs. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/108416)
+  - UDFs cannot call other UDFs or stored procedures from within their bodies. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/88198)
+  - The `setval` function cannot be resolved when used inside UDF bodies. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110860)
+  - DDL statements (e.g., `CREATE TABLE`, `CREATE INDEX`) are not allowed within UDFs or procedures. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/110080)
+  - The language does not support `OUT` and `INOUT` argument modes in UDF definitions. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/100405)
```

Comment:
This may be more of a PL/pgSQL than a function/procedure item; not sure

```suggestion
  - UDF and stored procedure definitions do not support `OUT` and `INOUT` argument modes. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/100405)
```